Author: Jens Petersen <petersen@redhat.com>
Source: FC14, patch from Mon Jul 17 2006, 1.4.4-24
Description: Fix various icon size problems
--- a/extras/panel/scim_panel_gtk.cpp
+++ b/extras/panel/scim_panel_gtk.cpp
@@ -111,7 +111,7 @@
 #define SCIM_PIN_DOWN_ICON_FILE     (SCIM_ICONDIR "/pin-down.png")
 
 #define TOOLBAR_ICON_SIZE                     16 
-#define TRAY_ICON_SIZE                        18
+#define TRAY_ICON_SIZE                        11
 #define LOOKUP_ICON_SIZE                      12
 
 /////////////////////////////////////////////////////////////////////////////
